src/components/Results/Results.tsx

```diff
@@ -33,21 +35,81 @@ const Results: React.FC<Props> = ({ measureFile, patientFile, htmls }) => {
   const results = useRecoilValue(resultsState);
   const detectedIssues = fhirpath.evaluate(results, 'Bundle.entry.resource.DetectedIssue');
 
-  return (
-    <Grid container xs={12} direction="column" spacing={2}>
-      <h2>Results:</h2>
-      <Grid container item xs={12} direction="row" justify="center" alignItems="center">
-        {outputType === 'measureReports' &&
-          calculationOptions.reportType === 'individual' &&
-          fhirpath.evaluate(results, 'MeasureReport.group').map((group: R4.IMeasureReport_Group) => {
-            const id = results ? fhirpath.evaluate(results, 'MeasureReport.subject.reference')[0].split('/') : '';
+  const style = {
+    minWidth: '25%'
+  };
```

Comment:
This is actually something you can add to the `makeStyles` call used above. Read more [here](https://material-ui.com/styles/basics/)

You can add a new entry in that object (under `highlightedMarkup`) that indicates the minWidth style that you want to apply, and access it off the `classes` object within the component